tests/playwright/src/model/pages/resource-details-page.ts

```diff
@@ -0,0 +1,45 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2025 Red Hat, Inc.
```

nit: FYI now it's 2025